Tiny.RestClient/TinyRestClient.cs

```diff
@@ -315,6 +330,8 @@ public IRequest DeleteRequest(string route = null)
            Request tinyRequest,
            CancellationToken cancellationToken)
         {
+            await ExecuteDynamicHeadersAsync(tinyRequest);
```

Comment:
ExecuteDynamicHeadersAsync must be executed on SendRequestAsync function just before to add headers